netmiko/plugin.spec.yaml

```diff
@@ -3,14 +3,15 @@ extension: plugin
 products: [insightconnect]
 name: netmiko
 title: Netmiko
-description: Netmiko is used to access network devices over SSH
-version: 1.0.1
+description: Access network devices over SSH
+version: 1.0.2
```

Comment:
I believe we want to leave the version numbers where they are right now because we never did a mass release of plugins after the last hackathon - should confirm this with @jschipp-r7

Reply:
"Yeah, it’s a major version bump if titles change"